docs/server/commands.rst

```diff
@@ -339,6 +342,25 @@ Available options:
 
   Default: ``name``.
 
+.. django-admin-option:: --only-emails
+
+  .. versionadded:: 2.8.0
+
+  Specifies to only output user names and emails. Users with no email are
+  skipped, thus anonymous users are not listed.
+
+.. django-admin-option:: --since
+
+  .. versionadded:: 2.8.0
+
+  Tells to only take into account contributions newer than the specified
```

Comment:
Only consider contributions since the specified date.

Avoid the programmer phrase Datetime, there is no such word.  Its just a date and the example makes it clear that you can specify a time.